python/paddle/fluid/tests/unittests/ir/inference/test_layer_norm_fuse_pass.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) 2021 PaddlePaddle Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Test for fusion of subgraph expressing layer normalization."""
+
+import unittest
+import numpy as np
+import paddle
+import paddle.fluid as fluid
+from inference_pass_test import InferencePassTest
+from paddle import enable_static
+from paddle.fluid.core import PassVersionChecker
+
+
+class LayerNormFusePassTest(InferencePassTest):
```

Comment:
You could add a conv before, otherwise the model cannot be saved because no parameters need to be saved, so load_combine fail

```
class LayerNormFusePassTest(InferencePassTest):
    def setUp(self):
        with fluid.program_guard(self.main_program, self.startup_program):
            data = fluid.data(name="data", shape=[-1, 64, 12, 12], dtype="float32")
            conv_out = fluid.layers.conv2d(
                input=data,
                num_filters=6,
                filter_size=6,
                groups=1,
                padding=[1, 1, 1, 1],
                bias_attr=False,
                act=None)
            sqr_pow = fluid.layers.fill_constant(
                shape=[1], value=2, dtype="float32")
            eps = fluid.layers.fill_constant(
                shape=[1], value=1e-5, dtype="float32")
            gamma = fluid.data(name="gamma", shape=[9], dtype="float32")
            beta = fluid.data(name="beta", shape=[9], dtype="float32")

            x_mean_out = fluid.layers.mean(conv_out)
            x_sub_mean_out = fluid.layers.elementwise_sub(conv_out, x_mean_out)
            x_sub_mean_sqr_out = fluid.layers.elementwise_pow(x_sub_mean_out,
                                                              sqr_pow)
            std_dev_out = fluid.layers.mean(x_sub_mean_sqr_out)
            std_dev_eps_out = fluid.layers.elementwise_add(std_dev_out, eps)
            std_dev_eps_sqrt_out = fluid.layers.sqrt(std_dev_eps_out)
            division_out = fluid.layers.elementwise_div(x_sub_mean_out,
                                                        std_dev_eps_sqrt_out)
            scale_out = fluid.layers.elementwise_mul(division_out, gamma)
            shift_out = fluid.layers.elementwise_add(scale_out, beta)

        self.feeds = {
            "data": np.random.random((3, 64, 12, 12)).astype("float32"),
            "gamma": np.random.random((9)).astype("float32"),
            "beta": np.random.random((9)).astype("float32"),
        }
        self.fetch_list = [shift_out]

    def test_check_output(self):
        use_gpu = False
        self.check_output_with_option(use_gpu)
        self.assertTrue(PassVersionChecker.IsCompatible("layer_norm_fuse_pass"))
```

Reply:
The solution is to use [`create_parameter`](https://github.com/PaddlePaddle/Paddle/blob/develop/python/paddle/fluid/layers/tensor.py#L76) for gamma and beta.